_pages/release-notes.md

```diff
@@ -10,6 +10,30 @@ title:  "Release Notes"
 tags: [alg-dev-getting-started, app-dev-getting-started]
 ---
 
+## Advanced Reporting for Governance
+
+The Advanced Reporting for Governance feature provides administrators the ability to view resource usage on the platform. Usage can be viewed at four different levels depending on your access permissions. If you’re an organization administrator, you can explore reporting at the organization and account level, and even down to the individual algorithm level. Cluster administrators are granted all of the above permissions, as well as being able to view reporting at the cluster level. We’ll start at the top level of the cluster roll-up.
```

Comment:
Cluster admins are the only ones that can add the base cost information. Not sure if you need this here.

Reply:
yeah i think that's probably good to mention. added in next commit